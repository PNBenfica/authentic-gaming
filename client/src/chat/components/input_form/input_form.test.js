import renderer from 'react-test-renderer';
import InputForm from './input_form';

it('renders correctly', () => {
  const tree = renderer
    .create(<InputForm />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});