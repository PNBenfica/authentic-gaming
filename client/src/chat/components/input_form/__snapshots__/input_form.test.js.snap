// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="box "
>
  <form
    onSubmit={[Function]}
  >
    <input
      autoComplete="off"
      className="input"
      name="name"
      onChange={[Function]}
      placeholder="Username"
      type="text"
      value=""
    />
    <input
      autoComplete="off"
      className="input"
      name="message"
      onChange={[Function]}
      placeholder="Message"
      type="text"
      value=""
    />
    <button
      className="btn disabled"
      type="submit"
    >
      Send
    </button>
  </form>
</div>
`;
